pkg/node/node.go

```diff
@@ -471,18 +473,28 @@ func addTaint(node *corev1.Node, nth Node, taintKey string, taintValue string, e
 		}
 
 		if err != nil {
-			log.Log().Msgf("Error while adding %v taint on node %v: %v", taintKey, node.Name, err)
+			log.Log().
```

Comment:
out of curiosity, what does this log look like compared to the one you have posted? I'm assuming you didn't wrap in `Err` for formatting reasons

log.Log().Err("Error while adding taint on node").Str("TaintKey", taintKey).  ...

Reply:
what do you mean by "compared to the one you have posted"? 

I thought wrapping our message in `Err()` would be misleading since the actual error is stored in `err`. So in the logs it would look like there are two errors when there's really only one. I removed `err` from the log statement here for a similar reason - since we're returning `err` it should be handled by the calling function and logging it in both places seemed redundant.

Reply:
Got it, I just meant comparing what you currently have in code now vs. the same line but wrapping the message with `Err()`